documentation/book/proc-downgrading-brokers-older-kafka.adoc

```diff
@@ -22,7 +20,7 @@ kind: Kafka
 spec:
   # ...
   kafka:
-    version: 2.1.0
+    version: 2.1.1
```

Comment:
So I guess the question is whether you want to be changing these version numbers every time there's a new Kafka version. Or whether it's acceptable for the documentation to describe the process in a more abstract way, not necessarily using the most current version numbers? If the former then I would suggest using asciidoc attributes is the correct way to deal with that.

Reply:
I think it's the former. I can create a new attribute named `LatestKafkaVersion` and set it to 2.1.1.

Reply:
Perhaps `LatestSupportedKafkaVersion` is better. This variable could have a different value in the upstream and downstream repositories.

Reply:
I do not think we need to be changing them every release ... we just need to use something supported everywhere.

Reply:
I think where we are now with this procedure is a mixture of using attributes `{KafkaVersionLower}`/`{KafkaVersionHigher}` and hard coded numbers `2.1.1`. This is almost bound result in problems because in the future the value of `{KafkaVersionLower}` will likely be changed without reviewing this procedure, resulting in a confusing mixture of version numbers, and/or it makes editing the procedure and error prone because the writer needs to take more into account. So I think this should either use all hard coded version numbers, or all attributes.

Reply:
Agreed. I've included the `{KafkaVersionLower}` and `{KafkaVersionHigher}` attributes in the code samples as well. There is no attribute for the message format version, so that will have to remain hard coded.

Reply:
I think that's going to make it super-easy to break the documentation by changing the attribute without changing the hard-coded version. Can't we add an attribute for the hard coded version?

Reply:
You're right. I've added two more attributes:
```
:LogMsgVersLower: 2.0
:LogMsgVersHigher: 2.1
```